screens/MainScreen.tsx

```diff
@@ -1,346 +1,404 @@
-    height: "4.19%",
-    top: "95.81%",
-    left: "0%",
-    right: "0%",
-    position: "absolute",
-    width: "100%",
-  },
-  border: {
-    width: "90.43%",
-    right: "9.57%",
-    borderRadius: 3,
-    borderStyle: "solid",
-    borderColor: "#fff",
-    borderWidth: 1,
-    opacity: 0.35,
-    left: "0%",
-    top: "0%",
-    height: "100%",
-    bottom: "0%",
-    position: "absolute",
-  },
-  capIcon: {
-    height: "35.29%",
-    width: "5.46%",
-    top: "32.35%",
-    bottom: "32.35%",
-    left: "94.54%",
-    opacity: 0.4,
-    right: "0%",
-  },
-  capacity: {
-    height: "64.71%",
-    width: "73.99%",
-    top: "17.65%",
-    right: "17.79%",
-    bottom: "17.65%",
-    left: "8.22%",
-    borderRadius: 1,
-  },
-  battery: {
-    height: "25.76%",
-    width: "6.49%",
-    top: "39.39%",
-    right: "3.82%",
-    bottom: "34.85%",
-    left: "89.69%",
-    position: "absolute",
-  },
-  wifiIcon: {
-    width: 15,
-    height: 11,
+    height: 34,
+    marginTop: 734,
+    width: 375,
   },
-  lte: {
-    top: 17,
-    left: 312,
-    fontSize: 11,
-    letterSpacing: -1,
-    fontWeight: "500",
-    fontFamily: FontFamily.sFProTextMedium,
-    width: 25,
-    height: 12,
-    display: "none",
+  deviceUi: {
+    left: -2,
+    width: 375,
+    top: 0,
   },
-  cellularIcon: {
-    height: "24.24%",
-    width: "4.53%",
-    top: "40.15%",
-    right: "17.07%",
-    bottom: "35.61%",
-    left: "78.4%",
+
+  untitled11: {
+    top: 737,
+    left: 119,
+    width: 115,
+    height: 32,
+    position: 'absolute',
+  },
+  container: {
+    borderRadius: 0,
+    shadowColor: 'rgba(7, 0, 83, 0.1)',
+    shadowOffset: {
+      width: 0,
+      height: 40,
+    },
+    shadowRadius: 44,
+    elevation: 44,
+    shadowOpacity: 1,
+    flex: 1,
+    overflow: 'hidden',
+    height: 812,
+    width: '100%',
+    backgroundColor: Color.white,
+    paddingVertical: 20,
   },
-  time: {
-    marginTop: -3.5,
-    top: "50%",
-    letterSpacing: 0,
-    fontWeight: "600",
-    fontFamily: FontFamily.sFProTextSemibold,
-    fontSize: FontSize.size_mini,
-    left: "0%",
-    width: "100%",
+  // Main Screen
+  frameScrollViewContent: {
+    flexDirection: 'column',
   },
-  barsStatusBarIphoneT: {
-    height: "47.73%",
-    width: "14.4%",
-    top: "15.91%",
-    right: "80%",
-    bottom: "36.36%",
-    left: "5.6%",
-    position: "absolute",
+  frameWrapperFlexBox: {
+    flex: 1,
+    overflow: 'hidden',
+  },
+  // bgPosition: {
+  //   width: 375,
+  //   left: 0,
+  //   position: 'absolute',
+  // },
+  // brandPosition: {
+  //   bottom: '0%',
+  //   left: '0%',
+  // },
+  // iconLayout: {
+  //   maxHeight: '100%',
+  //   maxWidth: '100%',
+  //   position: 'absolute',
+  //   overflow: 'hidden',
+  // },
+  // capacityBg: {
+  //   backgroundColor: Color.white,
+  //   position: 'absolute',
+  // },
+  // lteFlexBox: {
+  //   textAlign: 'center',
+  //   color: Color.white,
+  //   position: 'absolute',
+  // },
```

Comment:
Please Remove the commented code, remember not to push any unused code to the PR